11:40:56 - ERR - getRecords - (UndefinedColumn('no existe la columna «id_usuario»\nLINE 1: SELECT id_usuario, count(*) as valor FROM proyectop2.usuario...\n               ^\nHINT:  Probablemente quiera hacer referencia a la columna «usuarios.usuario».\n'),)
11:41:35 - ERR - getRecords - (GroupingError('la columna «usuarios.id» debe aparecer en la cláusula GROUP BY o ser usada en una función de agregación\nLINE 1: SELECT id, count(*) as valor FROM proyectop2.usuarios WHERE ...\n               ^\n'),)
11:43:13 - ERR - getRecords - (GroupingError('la columna «usuarios.id» debe aparecer en la cláusula GROUP BY o ser usada en una función de agregación\nLINE 1: SELECT id, count(*) as valor FROM proyectop2.usuarios WHERE ...\n               ^\n'),)
11:43:23 - ERR - getRecords - (GroupingError('la columna «usuarios.id» debe aparecer en la cláusula GROUP BY o ser usada en una función de agregación\nLINE 1: SELECT id, count(*) as valor FROM proyectop2.usuarios WHERE ...\n               ^\n'),)
11:43:44 - ERR - getRecords - (GroupingError('la columna «usuarios.id» debe aparecer en la cláusula GROUP BY o ser usada en una función de agregación\nLINE 1: SELECT proyectop2.usuarios.id, count(*) as valor FROM proyec...\n               ^\n'),)
11:44:06 - ERR - post - (TypeError("response_success() got an unexpected keyword argument 'extra_data'"),)
15:15:49 - ERR - getRecords - (SyntaxError('error de sintaxis en o cerca de «%»\nLINE 1: SELECT id FROM proyectop2.usuarios WHERE usuario = %s\n                                                           ^\n'),)
15:15:57 - ERR - getRecords - (SyntaxError('error de sintaxis en o cerca de «%»\nLINE 1: SELECT id FROM proyectop2.usuarios WHERE usuario = %s\n                                                           ^\n'),)
15:19:51 - ERR - getRecords - (SyntaxError('error de sintaxis en o cerca de «%»\nLINE 1: SELECT id FROM proyectop2.usuarios WHERE usuario = %s\n                                                           ^\n'),)
15:22:16 - ERR - getRecords - (SyntaxError('error de sintaxis en o cerca de «%»\nLINE 1: SELECT id FROM proyectop2.usuarios WHERE usuario = %s\n                                                           ^\n'),)
15:22:37 - ERR - getRecords - (SyntaxError('error de sintaxis en o cerca de «%»\nLINE 1: SELECT id FROM proyectop2.usuarios WHERE usuario = %s\n                                                           ^\n'),)
15:46:06 - ERR - getRecords - (TypeError("object of type 'int' has no len()"),)
15:48:26 - ERR - getRecords - (TypeError("object of type 'int' has no len()"),)
15:48:32 - ERR - getRecords - (TypeError("object of type 'int' has no len()"),)
15:50:10 - ERR - getRecords - (TypeError("object of type 'int' has no len()"),)
15:52:11 - ERR - getRecords - (NameError("name 'psycopg2' is not defined"),)
15:52:11 - ERR - get_user_id_by_username - (UnboundLocalError("cannot access local variable 'cursor' where it is not associated with a value"),)
17:30:47 - ERR - ExecuteNonQuery - (ForeignKeyViolation('inserción o actualización en la tabla «historias» viola la llave foránea «historias_id_usuario_fkey»\nDETAIL:  La llave (id_usuario)=(0) no está presente en la tabla «usuarios».\n'),)
17:31:00 - ERR - ExecuteNonQuery - (ForeignKeyViolation('inserción o actualización en la tabla «historias» viola la llave foránea «historias_id_usuario_fkey»\nDETAIL:  La llave (id_usuario)=(0) no está presente en la tabla «usuarios».\n'),)
